source/JABHandler.py

```diff
@@ -647,7 +647,11 @@ def getAccessibleTextSelectionInfo(self):
 		bridgeDll.getAccessibleTextSelectionInfo(self.vmID, self.accContext, byref(textSelectionInfo))
 		return textSelectionInfo
 
-	def _javaGetAccessibleTextRange(self, start, end):
+	def _javaGetAccessibleTextRange(self, start: int, end: int):
+		"""Helper method that performs the Java Access Bridge call to obtain the text of this object based on a (start, end) range
+		@param start: the starting position to get
+		@param end: the end position to get
+		@returns: the text as a string"""
```

Comment:
Please rewrite to use [our docstring standards](https://github.com/nvaccess/nvda/blob/master/projectDocs/dev/codingStandards.md#docstrings)
```suggestion

				:param start: The start index to get from, inclusive.
		:param end: The end index to fetch to, exclusive.
		:return: the text within the given indices as a string.
		"""
```